docs/rackhd/authentication.rst

```diff
@@ -0,0 +1,174 @@
+Accessing RackHD APIs with Authentication
+-----------------------------------------
+
+With authentication is enabled in config.json configure file for an endpoint,
+see `Setting up HTTP/HTTPS endpoint`_ and `Authentication`_, authentication is needed for
+accessing the API group associated with the specified endpoint.
+
+This section describes how to access APIs that need authentication.
+
+Setup endpoints to enable authentication
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Please refer to `Setting up HTTP/HTTPS endpoint`_ on how to setup endpoints. Simply put,
+the following endpoint configuration will be a good start.::
+
+    "httpEndpoints": [
+        {
+            "address": "0.0.0.0",
+            "port": 8443,
+            "httpsEnabled": true,
+            "proxiesEnabled": false,
+            "authEnabled": true,
+            "routers": "northbound-api-router"
+        },
+        {
+            "address": "172.31.128.1",
+            "port": 8080,
+            "httpsEnabled": false,
+            "proxiesEnabled": false,
+            "authEnabled": false,
+            "routers": "southbound-api-router"
+        }
+    ],
+
+The first endpoint represents a HTTPs service listening at port 8443 that serves northbound APIs, which are
+APIs that being called by users. Noticed that authEnabled is set to true means the authentication is needed
```

Comment:
Should just be "APIs being called by users", also "Note that authEnabled being set to true means that authentication is needed to access northbound APIs"